woocommerce/woocommerce-gutenberg-products-block
# Remove h1 from atomic title

This PR removes the option to have H1 as a level for the atomic Product Title block.


Fixes #1101 



### Screenshots

![image](https://user-images.githubusercontent.com/6165348/68761116-ade78d00-0613-11ea-9661-8ba9eb8a3d8b.png)



### How to test the changes in this Pull Request:

1. Create a new All Products block
2. Go to edit mode and select the title
3. See that H1 is not there anymore



### Changelog
- remove H1 svg
- remove ability to select H1
